lib/active_support/cache/memcached_store.rb

```diff
@@ -266,8 +244,14 @@ def expiration(options)
         expires_in
       end
 
-      def log_exception(e)
+      def handle_exceptions(return_value_on_error:, on_miss: return_value_on_error)
+        yield
+      rescue Memcached::NotFound, Memcached::ConnectionDataExists
+        on_miss
+      rescue Memcached::Error => e
+        raise unless @swallow_exceptions
         logger.warn("memcached error: #{e.class}: #{e.message}") if logger
```

Comment:
Nvm, misunderstood this. it's still logging if it's true and not logging (but raising) if it's false. That seems reasonable.